.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,12 +15,16 @@
 ## How Has This Been Tested?
 <!--- Include details of your testing environment, official tests or other methods -->
 
-## Followup Actions
+## Related Changes
 <!---
     Are the changes mandatory (breaking) or optional?
+
     What changes must a consumer of this repository make in order to utilise it?
-    Are there other issues or steps that need to happen once this PR is merged?
+
+    Are there other issues or steps that need to happen BEFORE this PR is merged? Start a list of these with the keyword "Prerequisite PRs:"
+
+    Are there other issues or steps that need to happen AFTER this PR is merged? Start a list of these with the keyword "Dependent PRs:"
```

Comment:
Can we remove the whitepsace? Its a lot to highlight and remove on every PR if we include whitespace. Can we make these more imperitive rather than asking the same question. Just to remove the sapce and make it obvious what needs to be done

Reply:
Agreed - it goes against convention and readability norms but templates lose their power the longer they are. 

If there's too much to read people (read: me) won't reliably read them. 

You also don't notice them if/when they get updated.